setup.cfg

```diff
@@ -82,6 +82,7 @@ commands =
     pytest {posargs:pylxd}
 
 [testenv:integration]
+passenv = *
```

Comment:
There must be a good reason for `tox` to scrape the environment variable, probably to avoid as much pollution/contamination as possible from the host running the test. As such, I'd prefer `passenv = LXD_*` or `passenv = LXD_TOKEN*`.

Reply:
I like the idea of using `LXD_*`, thanks! This would include things like LXD_INSPECT, LXD_DEBUG, etc. But I think none of those can pose any problems on the tests.